internal/integration/sqlite_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"testing"
 
 	"ariga.io/atlas/schema/schemaspec/schemahcl"
+	"ariga.io/atlas/sql/mysql"
```

Comment:
Why are you running mysql marhsla in sqlite test?

Reply:
good catch.thx